README.md

```diff
@@ -92,6 +92,10 @@ If you want to contribute to the project, please take a look at [CONTRIBUTING.md
   - [Hive/Hive2 Example](#hivehive2-example)
     - [Output](#output-12)
     - [Current limitations](#current-limitations-12)
+  - [Email Example](#email-example)
+    - [Output](#output-13)
+    - [Prerequisites](#prerequisites)
+    - [Known limitations](#known-limitations)
```

Comment:
I think we should decide wether to use "Current" or "Known" and use it consistently. I agree "Known limitations" is better but then we should likely make another PR fixing the "Current limitations" names in other places.

Reply:
Yes, I also had that feeling. I'll quickly prepare a PR.

Reply:
https://github.com/GoogleCloudPlatform/oozie-to-airflow/pull/291